modules/openxBidAdapter.md

```diff
@@ -15,14 +15,15 @@ Publishers are welcome to test the other adapter and give feedback. Please note
 # Bid Parameters
 ## Banner
 
-| Name | Scope | Type | Description | Example
-| ---- | ----- | ---- | ----------- | -------
-| `delDomain` or `platform` | required | String | OpenX delivery domain or platform id provided by your OpenX representative.  | "PUBLISHER-d.openx.net" or "555not5a-real-plat-form-id0123456789"
-| `unit` | required | String | OpenX ad unit ID provided by your OpenX representative. | "1611023122"
-| `customParams` | optional | Object | User-defined targeting key-value pairs. customParams applies to a specific unit. | `{key1: "v1", key2: ["v2","v3"]}`
-| `customFloor` | optional | Number | Minimum price in USD. customFloor applies to a specific unit. For example, use the following value to set a $1.50 floor: 1.50 <br/><br/> **WARNING:**<br/> Misuse of this parameter can impact revenue | 1.50
-| `doNotTrack` | optional | Boolean | Prevents advertiser from using data for this user. <br/><br/> **WARNING:**<br/> Request-level setting.  May impact revenue. | true
-| `coppa` | optional | Boolean | Enables Child's Online Privacy Protection Act (COPPA) regulations. | true
+| Name                            | Scope | Type | Description                                                                                                                                                                                                                                                                                                                                                                                                                        | Example
+|---------------------------------| ----- | ---- |------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------| -------
+| `delDomain` ~~or `platform`~~** | required | String | OpenX delivery domain provided by your OpenX representative.                                                                                                                                                                                                                                                                                                                                                                       | "PUBLISHER-d.openx.net"
```

Comment:
are we deprecating `platform`?

Reply:
true, I saw numerous ad requests with ui domain instead of del domain

Reply:
@kenan-gillet @marcin-wrobel-ox actually i believe @lambdarho was telling me that delDomain is actually preferred because using platform does not cookie sync as well